client/layout/color-scheme/index.jsx

```diff
@@ -48,13 +49,14 @@ export function refreshColorScheme( prevColorScheme, nextColorScheme ) {
 
 	if ( nextColorScheme ) {
 		classList.add( `is-${ nextColorScheme }` );
+	}
 
+	if ( config( 'theme_color_admin_color_scheme_override' ) ) {
```

Comment:
One thing I missed in the last review: we should use `config.isEnabled()` for retrieving the value of feature flags.

Reply:
Good catch, amended.